src/test/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervalsIntegrationTest.java

```diff
@@ -171,6 +175,81 @@ public void singleFileTest() {
         Assert.assertEquals(binsResult, binsExpected);
     }
 
+    @Test(dataProvider = "gridTestData")
+    public void gridTest(final int binLength, final int minBinLength, final String[] includeInterval, final String[] excludeIntervals, final String[] expectedIntervals) {
```

Comment:
Less picky about signatures (and everything else in) test code, but clean up if you like.